src/main/webapp/view/transcript/transcriptArea.js

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO: Update this import statement once TranscriptLineElement is in master.
+import {appendTextToList} from '/transcript.js';
+
+/** Loads the transcript lines onto the DOM. */
+export class TranscriptArea {
+  static #ENDPOINT_TRANSCRIPT = '/transcript';
+  static #TRANSCRIPT_CONTAINER = 'transcript-lines-container';
+  static #PARAM_ID = 'id';
+
+  #transcriptSeeker;
+  #eventController;
+
+  /**
+   * Creates an instance of `TranscriptArea` for loading
+   * transcript lines onto the DOM.
+   *
+   * @param eventController An event controller object that
+   *     that will be passed into a seekTranscript object.
+   */
+  constructor(eventController) {
+    console.log(eventController);
+    // TODO: Create a transcriptSeeker object with the
+    // eventController as the parameter.
+  }
+
+  /**
+   * Fetches the transcript lines from `ENDPOINT_TRANSCRIPT`.
+   *
+   * <p>This function assumes that the transcript lines have already
+   * been added to the datastore.
+   */
+  loadTranscript() {
```

Comment:
Would it be possible to make this `async` (and get rid of the `.then`s) so we can await it?  Just saying because the discussion will have to know when transcript is loaded before it can add indicators and making this async would allow .then or await in the calling method

Reply:
I think so. I made an issue to deal wit this later (#257) if that's okay.